.gitmodules

```
@@ -4,3 +4,6 @@
 [submodule "config/gdi-starter-kit/starter-kit-storage-and-interfaces"]
 	path = config/gdi-starter-kit/starter-kit-storage-and-interfaces
 	url = https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces.git
+[submodule "data"]
+	path = data
+	url = https://github.com/NBISweden/ejprd-data.git
```

```suggestion
        url = git@github.com:NBISweden/ejprd-data.git
```

I ran into problems when having the submodule as a https resource. The authentication does not work. It works if I use the ssh version, as above. Not sure if this would cause problems for others?

---

Got a confirmation that we can share subsets of the data so I’ve changes the repository to public access.